jekyll/_cci2_ja/introduction-to-the-circleci-web-ui.adoc

```diff
@@ -0,0 +1,181 @@
+---
```

Comment:
i think this file name should be:

```diff
- jekyll/_cci2_ja/introduction-to-the-circleci-web-ui.adoc
+ jekyll/_cci2_ja/introduction-to-the-circleci-web-app.adoc
```

based on https://github.com/circleci/circleci-docs/blob/master/jekyll/_cci2/introduction-to-the-circleci-web-app.adoc

Reply:
Thanks! This is because Ja site is one week behind and this PR does not cover [this change](https://github.com/circleci/circleci-docs/pull/6923/files). To avoid confusion when users switch languages, I'll change the file name and side nav this time (and the content next time).